### ISSUE TITLE
transfers.rsync() error when excluding tuple

### ISSUE THREAD

**Issue by kegan** [fabric/patchwork#20]
Using `transfers.rsync()` and excluding `tuple`:

```python
@task
def rsync(ctx):
    transfers.rsync(
        ctx,
        source=SOURCE,
        target=TARGET,
        exclude=('.git', '*.pyc')
    )
```

Generates the following error:

```
Traceback (most recent call last):
  File "/Users/kegan/.pyenv/versions/indoorcms-pyramid3/bin/fab", line 11, in <module>
    sys.exit(program.run())
  File "/Users/kegan/.pyenv/versions/3.6.5/envs/indoorcms-pyramid3/lib/python3.6/site-packages/invoke/program.py", line 332, in run
    self.execute()
  File "/Users/kegan/.pyenv/versions/3.6.5/envs/indoorcms-pyramid3/lib/python3.6/site-packages/invoke/program.py", line 480, in execute
    executor.execute(*self.tasks)
  File "/Users/kegan/.pyenv/versions/3.6.5/envs/indoorcms-pyramid3/lib/python3.6/site-packages/invoke/executor.py", line 133, in execute
    result = call.task(*args, **call.kwargs)
  File "/Users/kegan/.pyenv/versions/3.6.5/envs/indoorcms-pyramid3/lib/python3.6/site-packages/invoke/tasks.py", line 127, in __call__
    result = self.body(*args, **kwargs)
  File "/Users/kegan/projects/indoorcms-pyramid3/fabfile.py", line 50, in rsync
    delete=True
  File "/Users/kegan/.pyenv/versions/3.6.5/envs/indoorcms-pyramid3/lib/python3.6/site-packages/patchwork/transfers.py", line 116, in rsync
    'exclude': exclude_opts.format(exclusions),
IndexError: tuple index out of range
```

**Comment by kegan** [comment]
Submitted fix at pull request #21.

**Comment by bitprophet** [comment]
Let's roll into the PR.
